docs/src/pages/customization/theming/theming-de.md

```diff
@@ -63,7 +63,7 @@ Generieren Sie eine Themenbasis von den gegebenen Optionen.
 #### Parameter
 
 1. `options` (*Object*): Nimmt ein unvollständiges Themeobjekt auf und fügt die fehlenden Teile hinzu.
-2. `...args` (*Array*): Deep merge the arguments with the about to be returned theme.
+2. `...args` (*Object[]*): Deep merge the arguments with the about to be returned theme.
```

Comment:
The translations are managed in Crowdin, please revert https://github.com/mui-org/material-ui/blob/next/CONTRIBUTING.md#translations.

Reply:
Ok